Game/MyProject/Common.xcodeproj/xcuserdata/tamasb.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Bucket
+   type = "1"
```

Comment:
This should not be in the repository, alongside with some other files here. These are local settings that apply only to you.